Deploy/pack_poseEstimation_run.spec

```diff
@@ -0,0 +1,69 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+from PyInstaller.utils.hooks import collect_all
+from PyInstaller.utils.hooks import copy_metadata
+from PyInstaller.utils.hooks import collect_dynamic_libs
+
+
+block_cipher = None
+
+datas           = []
+binaries        = []
+hiddenimports   = []
+excludes        = []
+
+# datas += copy_metadata('deeplabcut', recursive=True)
+tmp_ret = collect_all('deeplabcut')
+datas += tmp_ret[0]; binaries += tmp_ret[1]; hiddenimports += tmp_ret[2]
+
+excludes = ["tensorflow","PySide6", "PyQT6", "IPython", "Markdown", "jupyter"]
```

Comment:
Should we also exclude PyQt5? I remember you were saying that there we conflicts between versions 5 and 6.
Make sure to exclude as many thing as we can, so we reduce the size of the final package